shared/test/test_net_sim_api.rb

```diff
@@ -41,14 +42,33 @@ def test_create_read_update_delete
 
     record_id = record_get_response['id'].to_i
 
-    assert_equal 8, update_record(record_id, {id: record_id, age: 8})['age']
-    record = read_records.first
+    assert_equal 8, update_record(record_id, {name: 'alice', id: record_id, age: 8})['age']
+    record = read_records(@url).first
     assert_equal 8, record['age']
+
+    # Test fetching starting from a minimum version.
+    record_create_response = create_record({name: 'bob'})
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `record_create_response`.